src/unit_model.js

```diff
@@ -0,0 +1,99 @@
+import { FlowchartItemMixin, InMemoryEntity } from "@exabyte-io/code.js/dist/entity";
+import { getUUID } from "@exabyte-io/code.js/dist/utils";
+import { mix } from "mixwith";
+
+import { MethodFactory } from "./methods/factory";
+import { buildUnitModelPath } from "./utils";
+
+export class UnitModel extends mix(InMemoryEntity).with(FlowchartItemMixin) {
+    static MethodFactory = MethodFactory;
+
+    constructor({ application, ...config }) {
+        const defaults = { flowchartId: getUUID() };
+        super({ ...defaults, ...config });
+    }
+
+    get tier1() {
+        return this.prop("tier1");
+    }
+
+    get tier2() {
+        return this.prop("tier2");
+    }
+
+    get tier3() {
+        return this.prop("tier3");
+    }
+
+    get type() {
+        return this.prop("type");
+    }
+
+    get subtype() {
+        return this.prop("subtype");
+    }
+
+    /**
+     *  Returns instance of a Method class.
+     */
+    get method() {
```

Comment:
should we use a similar convention as in Made.js/Material class, where when we call `this.basis` we return data and `this.Basis` a class instance?